stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -1172,15 +1167,13 @@ extension Set {
   ///   `false`.
   public static func == (lhs: Set<Element>, rhs: Set<Element>) -> Bool {
     switch (lhs._variantStorage, rhs._variantStorage) {
-    case (.native(let lhsNativeOwner), .native(let rhsNativeOwner)):
-      let lhsNative = lhsNativeOwner.nativeStorage
-      let rhsNative = rhsNativeOwner.nativeStorage
+    case (.native(let lhsNative), .native(let rhsNative)):
 
-      if lhsNativeOwner === rhsNativeOwner {
+      if lhsNative === rhsNative {
         return true
       }
 
-      if lhsNative.count != rhsNative.count {
+      if lhsNative._count != rhsNative._count {
```

Comment:
I think this was part of @austinzheng's patch. I'll admit, I don't really fully grok when _'s are considered appropriate or not. Especially for types which are already implementation detail. Seems fairly inconsistent?